src/server/admin/models.py

```diff
@@ -25,7 +25,7 @@ def _default_date_now():
 class User(Base):
     __tablename__ = "api_user"
     id = Column(Integer, primary_key=True, autoincrement=True)
-    roles = relationship("UserRole", secondary=association_table)
+    roles = relationship("UserRole", secondary=association_table, lazy="joined") # last arg does an eager load of this property from foreign tables
```

Comment:
Wow, that is awesome. Didn't know about that param.

Reply:
🌊 🏄